docs/content/preview/deploy/multi-dc/async-replication/async-transactional-failover.md

```diff
@@ -14,378 +14,318 @@ tags:
-                },
-                "snapshots": [
-                    {
-                        "id": "a83eca3e-e6a2-44f6-a9f2-f218d27f223c",
-                        "snapshot_time": "2023-06-08 17:56:46.109379"
-                    }
-                ]
-            }
-        ]
-    }
-    ```
+```sh
+./bin/yugabyted restore \
+    --database <database> \
+    --recover_to_point_in_time '<safe_time>'
+```
 
-1. Restore Standby (B) to the minimum latest consistent time using PITR.
-
-    ```sh
-    ./bin/yb-admin \
-        -master_addresses <standby_master_addresses> \
-        -certs_dir_name <cert_dir> \
-        restore_snapshot_schedule <schedule_id> "<safe_time>"
-    ```
-
-    Expect output similar to the following:
-
-    ```output.json
-    {
-        "snapshot_id": "034bdaa2-56fa-44fb-a6da-40b1d9c22b49",
-        "restoration_id": "e05e06d7-1766-412e-a364-8914691d84a3"
-    }
-    ```
-
-1. Verify that restoration completed successfully by running the following command. Repeat this step until the restore state is RESTORED.
-
-    ```sh
-    ./bin/yb-admin \
-        -master_addresses <standby_master_addresses> \
-        -certs_dir_name <cert_dir> \
-        list_snapshot_restorations
-    ```
-
-    Expect output similar to the following:
-
-    ```output.json
-    {
-        "restorations": [
-            {
-                "id": "a3fdc1c0-3e07-4607-91a7-1527b7b8d9ea",
-                "snapshot_id": "3ecbfc16-e2a5-43a3-bf0d-82e04e72be65",
-                "state": "RESTORED"
-            }
-        ]
-    }
+</div>
 
-1. Promote Standby (B) to the active role:
+<div id="local-pitr" class="tab-pane fade " role="tabpanel" aria-labelledby="local-pitr-tab">
 
-    ```sh
-    ./bin/yb-admin \
-        -master_addresses <standby_master_addresses> \
-        -certs_dir_name <cert_dir> \
-        change_xcluster_role ACTIVE
-    ```
+1. Find the Snapshot Schedule name for the database:
 
-    ```output
-    Changed role successfully
-    ```
+```sh
+./bin/yb-admin \
+    -master_addresses <B_master_addresses> \
+    list_snapshot_schedules
+```
 
-1. Delete the replication from the former Primary (A):
+Expect output similar to the following:
 
-    ```sh
-    ./bin/yb-admin \
-        -master_addresses <standby_master_addresses> \
-        delete_universe_replication <primary_universe_uuid>_<replication_name>
-    ```
+```output.json
+{
+    "schedules": [
+        {
+            "id": "034bdaa2-56fa-44fb-a6da-40b1d9c22b49",
+            "options": {
+                "filter": "ysql.dr_db2",
+                "interval": "1440 min",
+                "retention": "10080 min"
+            },
+            "snapshots": [
+                {
+                    "id": "a83eca3e-e6a2-44f6-a9f2-f218d27f223c",
+                    "snapshot_time": "2023-06-08 17:56:46.109379"
+                }
+            ]
+        }
+    ]
+}
+```
 
-1. Resume the application traffic on the new Primary (B).
+1. Restore to the `safe_time` of the database:
```

Comment:
this numbering doesn't seem to render properly btw, all the bullets in the preview show 1.

Reply:
@ddhodge can you please help.  I could not figure out why this is broken